src/interfaces/common.ts

```diff
@@ -8,3 +8,17 @@ export enum BooleanEnum {
 	true = "true",
 	false = "false",
 }
+
+// These variables were defined by DataPortal. Do not change their values as it will affect our metrics logs and dashboards.
+export enum AnalyticsVariablesEnum {
+	Product = "gitHubForJira",
+	GitHubConfigScreenEventName = "githubConfigurationScreen",
+	ScreenEvent = "screen",
+	UiEvent = "ui",
+	TrackEvent = "track",
+	OperationalEvent = "operational",
+	TraitEvent = "trait",
+	ProdEnv = "prod",
+	StgEnv = "stg",
+	DevEnv = "dev"
+}
```

Comment:
bad standard to have.  Enums should be succinct to a specific context. You should have multiple enums here.  Enums are also not to be used as constants, it's more for user/function inputs so you can help validate with compilation.